beacon-chain/sync/rpc_send_request.go

```diff
@@ -129,3 +134,65 @@ func SendBeaconBlocksByRootRequest(
 	}
 	return blocks, nil
 }
+
+func SendBlobSidecarByRoot(
+	ctx context.Context, tor blockchain.TemporalOracle, p2pApi p2p.P2P, pid peer.ID,
+	ctxMap ContextByteVersions, req *p2ptypes.BlobSidecarsByRootReq,
+) ([]*pb.BlobSidecar, error) {
+	topic, err := p2p.TopicFromMessage(p2p.BlobSidecarsByRootName, slots.ToEpoch(tor.CurrentSlot()))
+	if err != nil {
+		return nil, err
+	}
+	log.WithField("topic", topic).Debug("Sending blob sidecar request")
+	stream, err := p2pApi.Send(ctx, req, topic, pid)
```

Comment:
Should we validate request `req` beforehand? Mainly length check to not exceed `MaxRequestBlobsSidecars`